src/root/components/RiskImminentEventMap/GDACSEventMap/PointDetails/index.tsx

```diff
@@ -55,11 +55,23 @@ function PointDetails(props: PointDetailsProps) {
       end_date,
       event_details,
       alert_level,
+      hazard_type
     },
     populationExposure,
     onCloseButtonClick,
   } = props;
 
+  const severityDetail = React.useCallback(
```

Comment:
1. Let's use map to filter rather than using list with `includes`
2. Let's use early return